### PR TITLE
Gamified Exercise screen popup merge

### PR DESCRIPTION
**Pull request by benceisonline**
## Description

Added popup to exercise screen and made small adjustments to exercise screen frontend

## Changes

Updated exercise screen and directly related files, added a teeny tiny function to StorageService, and snapshots

## Related Issues

User info is not saved correctly:
https://github.com/Educado-App/educado-mobile/issues/111

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Any potential security implications have been considered and addressed.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)
Screenshot 2023-10-25 at 09 49 52

## If mobile/frontend pull request, what version of the backend is it stable, and running on? 

Dev / Staging

## Notes for Reviewers

<3

### CODE REVIEW

File: eml/constants/PopUpPhrases.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const successPhrases = [
+    `Bom trabalho, ${name}! 👍`,
+    `Legal, ${name}! 🙌`,
+    `Ótimo trabalho, ${name}! 🤙`,
+    `Maravilhoso 😊`,
+    `Uau! 😁`,
+    `Uhuuuu! 🤩`,
+    `Excelente! ✨`,
+    `Arrasou! 🎉`,
+    `Você está arrasando! 🔥`,
```
Comment (Vagn99) [inline eml/constants/PopUpPhrases.js:11]: Varme emojis 🔥

File: eml/services/StorageService.js [hunk 2]
```diff
@@ -7,8 +8,19 @@ const SUB_COURSE_LIST = '@subCourseList';
...
 const USER_ID = '@userId';
+const USER_INFO = '@userInfo';
 
-/** COURSE AND COURS LIST **/
+export const getUserInfo = async () => {
+  try {
+    const fetchedUserInfo = JSON.parse(await AsyncStorage.getItem(USER_INFO));
+    return fetchedUserInfo;
+  } catch (e) {
+    console.log("Couldn't fetch user info", e);
```
Comment (Vagn99) [inline eml/services/StorageService.js:18]: Console log 🥰

File: eml/constants/PopUpPhrases.js (hunk 1 above)
Comment (BenjaminVilladsen) [inline eml/constants/PopUpPhrases.js:11]: nice with emojis man

**Review by Vagn99 — CHANGES_REQUESTED** [review]
Lige en console log :P

**Review by BenjaminVilladsen — APPROVED** [review]

**Review by Vagn99 — APPROVED** [review]
Dayum it's good.

File: eml/config/keys.js
Comment (PrometheusBork) [inline eml/config/keys.js:1]: Should this be in front-end? Are you using this?

**Comment by Andazu** [comment]
Looks great, is there a reason to update every snapshot? I know we agreed to not make more snapshots in the future, but the ones that are here now should remain i think.

File: eml/config/keys.js
Comment (benceisonline) [reply eml/config/keys.js:1]: upsi, i delete

**Review by Andazu — COMMENTED** [review]
Looks great, is there a reason to update every snapshot? I know we agreed to not make more snapshots in the future, but the ones that are here now should remain i think.

File: eml/services/StorageService.js (hunk 2 above)
Comment (benceisonline) [reply eml/services/StorageService.js]: fixed

File: eml/config/keys.js
Comment (benceisonline) [reply eml/config/keys.js:1]: fixed

**Comment by PrometheusBork** [comment]
Looks good, I think the only problem is that you cannot navigate to the exercise screen, but that is of course a dependency from another group, so that will be implemented in the future. Good work!

**Review by PrometheusBork — APPROVED** [review]

**Pull request merged**